llvm/tools/llvm-gsymutil/llvm-gsymutil.cpp

```diff
@@ -95,6 +95,7 @@ static uint64_t SegmentSize;
 static bool Quiet;
 static std::vector<uint64_t> LookupAddresses;
 static bool LookupAddressesFromStdin;
+static bool StoreMergedFunctionInfo;
```

Comment:
We should probably initialize these inline just in case.
```
static bool StoreMergedFunctionInfo = false;
```
I know the others aren't doing it, but it would be good to start.